### PR TITLE
(#943) Added new hand-drawn tile: Pond

### PR DESCRIPTION
**Pull request by AppleC4T**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [x] Tried to [squash the commits][squash].
- [x] Added Screenshot / Path to tile to Indicate Location of Change in Game

### Changes done in this Pull Request

- View my work at https://pr-949-fossasia-labyrinth.surge.sh or at https://applec4t.github.io/labyrinth/

- Fixes https://github.com/fossasia/labyrinth/issues/943

### The problem I want to solve or the facility I want to improve is

Add a new hand-drawn tile to the game.

### My steps to solve it

Added new hand-drawn tile: Pond

### Screenshots, Location of Changes, Path to Tile

 - Path from the starting tile: `LEFT, LEFT, UP, UP, UP, UP, UP`

image

[squash]: https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

### CODE REVIEW

**Comment by abishekvashok** [comment]
I can merge this, but if you could intend this, I would be happy.

**Comment by AppleC4T** [comment]
@abishekvashok could you mark where would you want the indent? From my editor, everything seems aligned.

**Comment by abishekvashok** [comment]
Set tab to 4 spaces in your editor. Almost all lines are unintended.

**Comment by AppleC4T** [comment]
@abishekvashok is it ok now? If so I'll just squish the commits and it should be good to go.

**Comment by abishekvashok** [comment]
Still not properly intended. Please view on web.

**Comment by AppleC4T** [comment]
I think I know what you're talking about but always when I go into the editor (even the web version) it seems aligned. 
image

File: js/tiles.js [hunk 1]
```diff
@@ -458,6 +458,23 @@ const door = {
             this.ground = this.createImage("tiles/rooms/floor/caro.svg");
 		},
     }),
+	
+	drawnPond: Object.assign({}, OpenDoors, {
+		canEnterFromTheRight() {return false;},
+    	canLeaveToTheRight() {return false;},
```
Comment (abishekvashok) [inline js/tiles.js]: See the diff

**Comment by AppleC4T** [comment]
> I can merge this, but if you could intend this, I would be happy.

@abishekvashok sorry for the inconvenience, but I'm going to ask you to merge it, as I can't see the wrong indent. I tried to change and open in it few places and it still looks good on my end.

**Comment by AppleC4T** [comment]
Thanks!

**Pull request merged**